test/steps/setup.js

```diff
@@ -294,6 +297,11 @@ export function setUpCommandVerifyMessage() {
 	setUpInputStubs();
 }
 
+export function setUpCommandVerifyTransaction() {
+	setUpTransactionsStubs();
```

Comment:
What about crypto stubs?

Reply:
I think transaction utils is stubbed, crypto is not needed to be stubed